testsuite/MDAnalysisTests/lib/test_nsgrid.py

```diff
@@ -371,3 +375,26 @@ def test_issue_2670():
 
     # should return the one atom overlap
     assert len(ag2.select_atoms('around 0.0 resid 3')) == 1
+
+
+reason = "Turned off by default. The test can be enabled by setting the ENABLE_HIGH_MEM_UNIT_TESTS " \
+         "environment variable. Make sure you have at least 10GB of RAM."
```

Comment:
```suggestion
reason = ("Turned off by default. The test can be enabled by setting the ENABLE_HIGH_MEM_UNIT_TESTS "
          "environment variable. Make sure you have at least 10GB of RAM.")
```
Continuing strings using "\" breaks PEP8 I believe (or some other PEP), using implicit continuation using brackets is best where possible.

Reply:
Got it, done